web/themes/custom/votegov/templates/node/node--voter-guide--teaser.html.twig

```diff
@@ -4,14 +4,16 @@
   node.isSticky() ? 'node--sticky',
   not node.isPublished() ? 'node--unpublished',
   view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
-  'usa-card tablet:grid-col-6 tablet-lg:grid-col-4 desktop:grid-col-3 '
+  'usa-card tablet:grid-col-4 desktop-lg:grid-col-3'
 ] %}
 
 <article{{ attributes.addClass(classes) }}>
     <a href={{ url }}>
```

Comment:
let's add a `title` attribute on the link and set the value to be the `{{ label }}`